Yubico.YubiKey/src/Yubico/YubiKey/YubiKeyDeviceInfo.cs

```diff
@@ -361,6 +380,15 @@ internal YubiKeyDeviceInfo Merge(YubiKeyDeviceInfo? second)
 
         /// <inheritdoc/>
         public override string ToString() =>
-            $"{nameof(AvailableUsbCapabilities)}: {AvailableUsbCapabilities}, {nameof(EnabledUsbCapabilities)}: {EnabledUsbCapabilities}, {nameof(AvailableNfcCapabilities)}: {AvailableNfcCapabilities}, {nameof(EnabledNfcCapabilities)}: {EnabledNfcCapabilities}, {nameof(AutoEjectTimeout)}: {AutoEjectTimeout}, {nameof(DeviceFlags)}: {DeviceFlags}, {nameof(ConfigurationLocked)}: {ConfigurationLocked}";
+            $"{nameof(AvailableUsbCapabilities)}: {AvailableUsbCapabilities}, " +
+            $"{nameof(EnabledUsbCapabilities)}: {EnabledUsbCapabilities}, " +
+            $"{nameof(AvailableNfcCapabilities)}: {AvailableNfcCapabilities}, " +
+            $"{nameof(EnabledNfcCapabilities)}: {EnabledNfcCapabilities}, " +
+            $"{nameof(IsNfcRestricted)}: {IsNfcRestricted}, " +
+            $"{nameof(AutoEjectTimeout)}: {AutoEjectTimeout}, " +
+            $"{nameof(DeviceFlags)}: {DeviceFlags}, " +
+            $"{nameof(ConfigurationLocked)}: {ConfigurationLocked}, " +
+            $"{nameof(PartNumber)}: {PartNumber}, " +
+            $"{nameof(IsPinComplexityEnabled)}: {IsPinComplexityEnabled}";
```

Comment:
Several properties added in this PR are missing here.

Reply:
I know.. I asked Greg and he said as long as we can identity a YK it should be find with the fields currently set. The best way to solve this would be to have it dynamically generated using reflection looking at public properties of the class. And at that point I think it is redundant because that's usually what I inspect in the IDE's anyway. I can add this to the backlog and have this slip through for now :pray: